connector/connect/common/src/main/protobuf/spark/connect/commands.proto

```diff
@@ -31,6 +31,7 @@ option java_package = "org.apache.spark.connect.proto";
 // produce a relational result.
 message Command {
   oneof command_type {
+    CommonInlineUserDefinedFunction register_function = 1;
```

Comment:
this was just removed in https://github.com/apache/spark/commit/9fbdb2dc095131ebd5be41f61e9481574a40be4d

is it still needed?

Reply:
Yeah, this is a replacement of reusing our existing Python protobuf message.

Reply:
oh, they are different protos.